python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1335,8 +1358,23 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
 
             for d, detector in enumerate(visitSummary["id"]):
                 mapName = f"{visit}/{detector}"
-
-                mapElements = wcsf.mapCollection.orderAtoms(f"{mapName}/base")
+                if mapName in wcsf.mapCollection.allMapNames():
+                    mapElements = wcsf.mapCollection.orderAtoms(f"{mapName}/base")
+                else:
+                    # This extension was not fit, but try to recover WCS
```

Comment:
Assuming "extension" means "detector/visit": if a detector/visit wasn't fit, how could there be a WCS?

Reply:
Yes, "extension" means "detector/visit". The detector part of the model is fixed by the sources from this detector on other visits, and the visit part of the model is fixed using the other detectors for the visit. Actually, this makes me realize I need to add a check so that this doesn't fail when there is only one detector.

Reply:
Scratch the part about what happens when there is only one detector. That will be caught at line 1354.